cmd/consensusGenome/consensusGenome.go

```diff
@@ -149,18 +149,25 @@ func validateCommonArgs() error {
 	if technology == "" {
 		return errors.New("missing required argument: sequencing-platform")
 	}
+	if technology != "Illumina" && (referenceAccession != "" || referenceFasta != "" || primerBed != "") {
```

Comment:
non-blocking, it seems like all of these conditions would be more readable if we split this function up into smaller functions with more descriptive names at some point, but definitely not needed for this PR.

Reply:
Yeah I agree this has become a bit of a mess little by little. Long term we want to split this out into general-cg and sars-cov2-cg because a lot of the conditions split on that and it will simplify things a lot. This just gets things usable for now and hopefully the test cases keep things accurate.